static/styles/_components/_side-panel.scss

```diff
@@ -23,24 +23,144 @@
 }
 
 .side-panel {
-  padding: 2rem;
-  top: 0;
+  background: $lightest-gray;
+  border-color: $light-gray;
+  border-style: solid;
+  border-bottom-width: 1px;
+  border-top-width: 0;
   bottom: 0;
+  //padding: 1rem;
+  position: absolute;
+  top: 0;
+  width: 100%;
+
+  h4 {
+    margin-top: 1em;
+  }
+
+  &.side-panel--open {
+    position: relative;
+
+    .side-panel__toggle {
+      .icon {
+        &:before {
+          content: "\e648"; // Up
+        }
+      }
+
+      @include media($large) {
+        .icon {
+          &:before {
+            content: "\e64a"; // Left
+          }
+        }
+      }
+
+      &:after {
+        content: "Hide filters";
+      }
+
+      &:hover, &:focus {
+        &:after {
+          content: "Hide filters" !important;
+        }
+      }
+    }
+  }
+
+  .side-panel__toggle {
+    @include transform(translateX(-50%));
+    border-color: $light-gray;
+    left: 50%;
+    padding: 1rem;
+    position: absolute;
+    z-index: 100;
+
+    &:hover {
+      background: $lightest-gray;
+    }
+
+    &:after {
+      content: "Show filters";
+    }
+
+    .icon {
+      display: inline-block;
+      padding-right: .3em;
+
+      &:before {
+        content: "\e64b"; // Down
+      }
+    }
+
+    @include media($large) {
+      @include transform(translateX(0));
+
+      &:after {
+        content: "" !important;
+      }
+
+      &:hover, &:focus {
+        &:after {
+          content: "Show filters" !important;
+        }
+      }
+
+      .icon {
+        display: inline-block;
+        padding-right: .3em;
+
+        &:before {
+          content: "\e649"; // Right
+        }
+      }
+    }
+  }
+
+
 }
 
 .side-panel--left {
   @include clearfix();
-  display: none;
-  background: $lightest-gray;
-  width: 100%;
+  @include transition(margin-left .5s);
+  border-right-width: 1px;
+  height: 0;
+  margin-right: 3%;
+
+  @include media($large) {
+    height: auto;
+    float: left;
+    margin-left: -17%;
+    position: static;
+    width: 17%;
+
+    .side-panel__box {
+      position: relative;
+    }
+
+    .side-panel__toggle.side-panel__toggle {
```

Comment:
Unfortunately, no. It's a big of a specificity hack to get around having to use `!important`.